gormzap.go

```diff
@@ -122,3 +119,57 @@ func formatNumbered(index int) string {
 func formatQuestioned(index int) string {
 	return "?"
 }
+
+func formatValue(value interface{}) string {
+	indirectValue := reflect.Indirect(reflect.ValueOf(value))
+	if !indirectValue.IsValid() {
+		return "NULL"
+	}
+
+	value = indirectValue.Interface()
+
+	switch v := value.(type) {
+	case time.Time:
+		return fmt.Sprintf("'%v'", v.Format("2006-01-02 15:04:05"))
+	case []byte:
+		s := string(v)
+		if isPrintable(s) {
+			return fmt.Sprintf("'%s'", s)
+		}
+		if h, ok := isWeirdHex(v); ok {
+			return fmt.Sprintf("'%s'", h)
+		}
+		return "'<binary>'"
+	case int, int8, int16, int32, int64,
+		uint, uint8, uint16, uint32, uint64:
+		return fmt.Sprintf("%d", v)
+	case driver.Valuer:
+		if dv, err := v.Value(); err == nil && dv != nil {
+			return formatValue(dv)
+		} else {
```

Comment:
if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)